### PR TITLE
Update documentation scripts for latest directory structure

### PR DESCRIPTION
**Pull request by berquist**
This fixes the docs for our updated directory structure. The production version is now new enough that a lot of the import try/except for different versions in `coverage.py` could be removed.

The indentation in `coverage` and `attributes` was also bumped a level.

### CODE REVIEW

File: sphinx/attributes.py [hunk 1]
```diff
@@ -2,81 +2,67 @@
...
-    """Make sure we are importing code from a subdirectory, which should exist
-    and should have been updated just before running this script. Note that
-    this script does not assume any version in the module and just takes
-    what it finds... so an appropriate checkout should be done first."""
-    if cclib.__file__[:len(os.getcwd())] != os.getcwd():
-        print("Do not seem to be importing from current directory")
-        sys.exit(1)
-
-
-if __name__ == "__main__":
```
Comment (langner) [inline sphinx/attributes.py]: I would prefer to leave main in place. Maybe put the code below into some helper function(s)?

File: sphinx/coverage.py [hunk 2]
```diff
@@ -5,139 +5,97 @@
 import os
 import sys
 
-from attributes import check_cclib
-
-
-if __name__ == "__main__":
```
Comment (langner) [inline sphinx/coverage.py]: Ditto.

**Comment by berquist** [comment]
Everything seems to be ok formatting-wise, except for the `transprop` description being "all absorption and emission spectra (dictionary {name:". This must be due to it having multiple lines in the ccData docstring.

I think we should avoid making any other changes to the docs until https://github.com/cclib/cclib.github.io/issues/32 is done. I know how to get a simple version of it working, if I find time over the next few days.

**Comment by langner** [comment]
Thanks!

**Pull request merged**